projects/plugins/protect/src/js/components/firewall-header/index.jsx

```diff
@@ -0,0 +1,138 @@
+import { AdminSectionHero, Container, Col, Text, H3, Button } from '@automattic/jetpack-components';
+import { useProductCheckoutWorkflow } from '@automattic/jetpack-connection';
+import { Spinner, Popover } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { Icon, help } from '@wordpress/icons';
+import classnames from 'classnames';
+import React, { useState, useCallback } from 'react';
+import useAnalyticsTracks from '../../hooks/use-analytics-tracks';
+import { JETPACK_SCAN } from '../admin-page';
+import styles from './styles.module.scss';
+
+const FirewallHeader = ( { status, hasRequiredPlan } ) => {
+	const { adminUrl } = window.jetpackProtectInitialState || {};
+
+	const { run } = useProductCheckoutWorkflow( {
+		productSlug: JETPACK_SCAN,
+		redirectUrl: adminUrl,
+	} );
+
+	const { recordEventHandler } = useAnalyticsTracks();
+	const getScan = recordEventHandler( 'jetpack_protect_waf_header_get_scan_link_click', run );
+
+	const [ showPopover, setShowPopover ] = useState( false );
+
+	const handleEnter = useCallback( () => {
+		setShowPopover( true );
+	}, [] );
+
+	const handleOut = useCallback( () => {
+		setShowPopover( false );
+	}, [] );
+
+	if ( 'on' === status ) {
+		return (
+			<AdminSectionHero>
+				<Container
+					className={ styles[ 'firewall-header' ] }
+					horizontalSpacing={ 6 }
+					horizontalGap={ 0 }
+				>
+					<Col>
+						<Text className={ classnames( styles.status, styles.active ) } variant={ 'label' }>
+							{ __( 'Active', 'jetpack-protect' ) }
+						</Text>
+						<H3 className={ styles[ 'firewall-heading' ] } mb={ 2 } mt={ 2 }>
+							{ __( 'Automatic firewall is on', 'jetpack-protect' ) }
+						</H3>
+					</Col>
+					<Col>
+						<div className={ styles[ 'stat-card-wrapper' ] }></div>
+					</Col>
+				</Container>
+			</AdminSectionHero>
+		);
+	}
+
+	if ( 'off' === status ) {
+		return (
+			<AdminSectionHero>
+				<Container
+					className={ styles[ 'firewall-header' ] }
+					horizontalSpacing={ 6 }
+					horizontalGap={ 0 }
+				>
+					<Col>
+						<Text className={ styles.status } variant={ 'label' }>
+							{ __( 'Inactive', 'jetpack-protect' ) }
+						</Text>
+						<H3 className={ styles[ 'firewall-heading' ] } mb={ 2 } mt={ 2 }>
+							{ __( 'Automatic firewall is off', 'jetpack-protect' ) }
+						</H3>
+						{ ! hasRequiredPlan && (
+							<>
+								<div className={ styles[ 'manual-rules-notice' ] }>
+									<Text variant={ 'body-small' } weight={ 600 }>
```

Comment:
```suggestion
									<Text weight={ 600 }>
```
Text size should be `16px`